packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature

```diff
@@ -7,14 +7,14 @@ Feature: Sign In with Force New Password flow
   Background:
     Given I'm running the example "ui/components/authenticator/sign-in-with-phone"
 
-  @react @vue @skip
+  @next @react @vue @skip
```

Comment:
What do you think about adding a `//` comment like we would a TODO to know what's needed for this to work?

```suggestion
  // The `FORCE_CHANGE_NUMBER` env variable isn't in the correct state to test this
  @skip @angular @next @react @vue
```